examples/00-load/create-sphere.py

```diff
@@ -0,0 +1,239 @@
+"""
+.. _create_sphere:
+
+Create Sphere Mesh Multiple Ways
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example shows how to create meshes in different ways.
+
+"""
+# sphinx_gallery_thumbnail_number = 5
+
+# %%
+from __future__ import annotations
+
+import numpy as np
+
+import pyvista as pv
+
+# %%
+# Simple Sphere
+# ~~~~~~~~~~~~~
+# The quickest method to get a Sphere mesh is to use :func:`pyvista.Sphere`.
+
+mesh = pv.Sphere()
+mesh.plot(show_edges=True)
+
+# %%
+# This gives an :class:`pyvista.PolyData` mesh, i.e. a 2D surface. In this case, it is watertight
+# and encloses a volume.
```

Comment:
Is watertight the same as `is_manifold` here? https://dev.pyvista.org/api/core/_autosummary/pyvista.polydata.is_manifold#pyvista.PolyData.is_manifold
Not sure if it's helpful to cross-ref this, though this property itself is a bit ambiguously defined, see
https://github.com/pyvista/pyvista/issues/4736

Is it overkill to also show that its `volume` is zero (since it's a 2D surface)?

Reply:
I think it does mean the same thing. Good point, I will update the language. Linking is a good idea too, but I'm on the fence based on the following:

The only 2D surface that has a valid volume (in vtk/PyVista) is a manifold PolyData mesh. `pv.Sphere()` has a volume, but `pv.Sphere().cast_to_unstructured_grid()` does not have a volume.  The `StructuredGrid` example that is later also does not have a volume.  `is_manifold` also only exists for PolyData meshes.  These subtleties are important but are better IMO in an example that compares the different mesh types rather than an example that shows different ways to form those meshes.

Reply:
Ok, I'm fine with not including any links or comparing/discussing volume. These were partly just my own curiosities.